explorer/client/src/components/search/Search.tsx

```diff
@@ -90,11 +90,11 @@ function Search() {
             setInput(e.currentTarget.value),
         [setInput]
     );
-    const handleCategoryChange = useCallback(
+    /* const handleCategoryChange = useCallback(
```

Comment:
This should be removed rather than commented out. Also, as `category` will now always be `'all'`:

* `getPlaceholderText` (line 18) should be removed as the text will always read as 'Search by ID'.
* `isInputValid` (line 30) should be simplified to just the `'all'` case.
* The useState for setting and reading the category should be removed as the category will always be `'all'` (see line 50).
* Lines 78-82 should be removed as this is now redundant code.